.github/workflows/preview-links.yml

```diff
@@ -35,10 +35,7 @@ jobs:
           PR_URL: ${{ github.event.pull_request.url }}
         run: |
           yarn create-preview-links "$PR_URL" "$PR_NUMBER"
-
       - name: Comment
-        if: steps.links.outputs.comment != ''
-        env:
-          COMMENT: ${{ steps.links.outputs.comment }}
+        if: env.comment != ''
         run: |
-          gh pr comment "$PR_NUMBER" --body "$COMMENT"
+          gh pr comment "$PR_NUMBER" --body "env.comment"
```

Comment:
I'd be a little surprised if this is working as is, [the docs](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-writing-an-environment-variable-to-github_env), show that you need to do a "${{ }}" to access the values.
We'll also want to pull this value into a string variable before using it directly in the script. See my other comment in the `validate_packs.yml` file

Reply:
I saw that in the docs and was a little curious, because we werent using it in the `gh pr comment` command in that way.  i see now though, its because its a string and not env var! i updated this and `validate_packs.yml` to keep the env declaration and use the string we set